### PR TITLE
Dev-tools: Configure for subdirectory usage

### PR DESCRIPTION
**Pull request by rebeccahum**
To play nice with https://github.com/Automattic/vip-cli/pull/1334

### CODE REVIEW

**Review by pschoffer — CHANGES_REQUESTED** [review]
Looks good.

We probably should also bump the dev-tools verision [here](https://github.com/Automattic/vip-container-images/blob/master/.github/workflows/dev-tools.yml#L57) that way existing dev-envs that send in the positional arguments will continue to work fine. (after that in you CLI PR - you can change [this line](https://github.com/Automattic/vip-cli/blob/develop/assets/dev-env.lando.template.yml.ejs#L17) to use your new dev-tools moving forward)


Also using `getopt` might be beneficial for doing that argument parsing, but it would not help that much on the full name options, and I think those are good as they sort of self-document. So probably just ignore this `getopt` point:)

**Comment by sjinks** [comment]
We can check whether the first command line parameter starts with `--`. If it does, we use `getopt`. If it doesn't, we fall back to the old scheme with positional parameters.

@pschoffer, @rebeccahum what do you think?

File: dev-tools/setup.sh [hunk 1]
```diff
@@ -1,15 +1,51 @@
 #!/bin/sh
 
 if [ $# -lt 4 ]; then
```
Comment (sjinks) [inline dev-tools/setup.sh:3]: I am afraid this won't work as expected :-( We now have 8 required parameters: `--host`, `<host>`, `--user`, `<user>`, `--domain`, `<domain>`, `--title`, `<title>`.

**Comment by rebeccahum** [comment]
@sjinks @pschoffer My only question now is, what if someone has a new version of VIP-CLI with an old container image where it attempts to pass in the long form arguments where the script doesn't know them yet?

**Comment by sjinks** [comment]
@rebeccahum I would say we need to wait for a couple of weeks until we merge the vip-cli PR.

**Review by sjinks — APPROVED** [review]

**Pull request merged**